packages/shared/src/components/comments/CommentInputOrModal.tsx

```diff
@@ -42,7 +44,7 @@ export default function CommentInputOrModal({
     <WriteCommentContext.Provider
       value={{ mutateComment: mutateCommentResult }}
     >
-      <CommentMarkdownInput {...props} className={className.input} />
+      <CommentMarkdownInput {...props} className={className.input} onRequestClose={onRequestClose}/>
```

Comment:
Now, for this one. We can just use the `onClose` 🥳 
```suggestion
      <CommentMarkdownInput {...props} className={className.input} onRequestClose={onClose}/>
```